src/navigation/stacks/main/index.tsx

```diff
@@ -59,6 +82,32 @@ const MainStack = () => {
       />
     </Tab.Navigator>
   );
+}
+
+const ProductStack = () => {
+  return (
+    <StackNav.Navigator screenOptions={{ headerShown: false }}>
+      <StackNav.Screen name={MainStackScreens.Products} component={ProductsScreen} />
+      <StackNav.Screen name={MainStackScreens.Detail} component={DetailScreen} />
+    </StackNav.Navigator>
+  );
+};
+
+const HomeStack = () => {
+  return (
+    <StackNav.Navigator screenOptions={{ headerShown: false }}>
+      <StackNav.Screen name={MainStackScreens.Home} component={HomeScreen} />
+      <StackNav.Screen name={MainStackScreens.Detail} component={DetailScreen} />
```

Comment:
You're including this screen twice, in `ProductsStack` and here, is this needed?

Reply:
It was needed. The app didnt know where to go if I didnt add it there